core-java-modules/core-java-collections-4/src/test/java/com/baeldung/collections/priorityqueue/PriorityQueueComparatorUnitTest.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.collections.priorityqueue;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.PriorityQueue;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class PriorityQueueComparatorUnitTest {
+
+    @Test
+    public void givenIntegerQueue_defaultComparator_followsNaturalOrdering() {
```

Comment:
jupiter tests could be default
```suggestion
    void givenIntegerQueue_defaultComparator_followsNaturalOrdering() {
```